.github/workflows/ca-clone-hsm-test.yml

```diff
@@ -162,6 +162,22 @@ jobs:
 
           docker exec primary pki-server cert-export ca_signing \
               --cert-file ${SHARED}/ca_signing.crt
+          # export CA signing CSR
+          docker exec primary pki-server cert-export ca_signing \
+              --csr-file ${SHARED}/ca_signing.csr
+
+          # export CA OCSP signing CSR
+          docker exec primary pki-server cert-export ca_ocsp_signing \
+              --csr-file ${SHARED}/ca_ocsp_signing.csr
+
+          # export CA audit signing CSR
+          docker exec primary pki-server cert-export ca_audit_signing \
+              --csr-file ${SHARED}/ca_audit_signing.csr
+
+          # export subsystem CSR
+          docker exec primary pki-server cert-export subsystem \
+              --csr-file ${SHARED}/subsystem.csr
+
```

Comment:
Once the loop for importing the CSRs from the master is restored, we can restore the tests too, so the secondary CA is used to test cloning without CSR files, and the tertiary CA is used to test cloning with CSR files.